packages/content-format/src/nodes/HeadingNode.ts

```diff
@@ -1,14 +1,21 @@
 import { ComposedElement } from '../ComposedElement';
 import type { Node } from '../Node';
 
-export interface HeadingNode<Child extends Node> extends ComposedElement<`${HeadingNode.Type}`, Child> {}
+export interface HeadingNode<Child extends Node> extends ComposedElement<`${HeadingNode.Type}`, Child> {
+    role?: HeadingNode.Role;
```

Comment:
I recommend stringifying the role enum to allow using plain string 'title' and 'subtitle' as role values.

```suggestion
    role?: `${HeadingNode.Role}`;
```